### DISCUSSION TITLE
Customizing via componentMapper

### DISCUSSION THREAD

**Discussion by jinx0001** [data-driven-forms/react-forms#1318] [Need help]
Hey,

I was wondering if there is a more detailed documentation about the global properties that the component mapper can take for customizing the form components like text fields and so on?
E.g. I did not find anything for changing the label colors of a MUI text field, because there is only the FormFieldGridProps, which (to my understanding) only targets the Grid and not the text field itself. 
Is this possible?

**Comment by Hyperkid123** [comment]
@jinx0001 the schema is mapped to the props. So any prop accepted by the TextField can be used in the schema/component mapper config.

EDIT: Here is a sample with global placeholder and variant props: https://codesandbox.io/s/hardcore-boyd-t1hszi?file=/src/App.js
